lsix

```diff
@@ -232,7 +232,7 @@ main() {
 	# While we still have images to process...
 	onerow=()
 	goal=$(($# - numtiles)) # How many tiles left after this row
-	while [ $# -gt 0  -a  $# -gt $goal ]; do
+	while [ $# -gt 0 ] && [ $# -gt $goal ]; do
```

Comment:
I don't think it is actually ambiguous since we're using bash. If I had to rewrite it, I'd prefer `[[  $# -gt 0  &&  $# -gt $goal  ]]`

Reply:
I changed it.